Makefile

```diff
@@ -20,12 +20,10 @@ endef
 $(SERVICES):
 	$(call compile_service,$(@))
 
+
+# Install all non-WASM executables from the build directory to GOBIN with 'propeller-' prefix
 install:
-	for file in $(BUILD_DIR)/*; do \
-		if [[ ! "$$file" =~ \.wasm$$ ]]; then \
-			cp "$$file" $(GOBIN)/propeller-`basename "$$file"`; \
-		fi \
-	done
+	$(foreach f,$(wildcard $(BUILD_DIR)/*[!.wasm]),cp $(f) $(patsubst $(BUILD_DIR)/%,$(GOBIN)/propeller-%,$(f));)
```

Comment:
Would it be possible, in order to simplify handling of binaries, to have a subdir `wasm` within `build` dir, and put all Wasm binaries there. I guess these binaries are coming from examples, right? If really needed you can also add `build/bin` and `build/wasm`, but I do not think that we need `build/bin` (as most of the time we'll need just Propeller services (and not examples), and they can live directly in the top `build` dir, for convenience), but if this simplifies copying, then please go ahead and make both `build/bin` and `build/wasm`.

Reply:
> Would it be possible, in order to simplify handling of binaries, to have a subdir `wasm` within `build` dir, and put all Wasm binaries there. I guess these binaries are coming from examples, right? If really needed you can also add `build/bin` and `build/wasm`, but I do not think that we need `build/bin` (as most of the time we'll need just Propeller services (and not examples), and they can live directly in the top `build` dir, for convenience), but if this simplifies copying, then please go ahead and make both `build/bin` and `build/wasm`.

We can implement this .